scripts/next_version.sh

```diff
@@ -78,8 +78,7 @@ fi
 
 if [[ "${EXEC_ALPHA}" == true ]]
 then
-    ISH_KEY=$(git rev-parse HEAD)
-    NEXT_VERSION=${NEXT_VERSION}-${ISH_KEY}
+    NEXT_VERSION=${NEXT_VERSION}-${TRAVIS_BUILD_NUMBER}
```

Comment:
Is this script only used in the CI? `TRAVIS_BUILD_NUMBER` is provided only in CI, if it is used only in the CI, we might consider to move it under `scripts/travis`, so no one can have the false expectation to work locally.

Reply:
If not, I suggest this `TRAVIS_BUILD_NUMBER` to have some more generic name, and be an input param for this script.

Reply:
only CI